BlazarUI/app/stylus/components/module-modal.styl

```diff
@@ -0,0 +1,12 @@
+.module-modal--container
+  height 200px
+
+.module-modal--modules
+  float left
```

Comment:
i would just keep this simple and use the bootstrap grid, no need for custom css